src/main/java/friday/FridayException.java

```diff
@@ -0,0 +1,10 @@
+package friday;
+
+/**
+ * Represents an exception specific to the Friday application.
+ */
+public class FridayException extends Exception {
+    public FridayException(String message) {
+        super(message);
+    }
```

Comment:
Should the constructor have some Javadoc explaining what the parameters mean?

```suggestion
/**
 * Constructs an instance of {@code FridayException}.
 * 
 * @param message Exception message containing details of the error
 */
public FridayException(String message) {
    super(message);
}
```